cobigen/cobigen-core-parent/cobigen-core/src/main/java/com/devonfw/cobigen/impl/externalprocess/ExternalProcessHandler.java

```diff
@@ -445,6 +409,8 @@ public boolean initializeConnection() {
                 // Just check correct port acquisition
                 if (acquirePort()) {
                     return true;
+                } else {
+                    continue;
```

Comment:
it's ok, but actually not needed as the look will anyhow continue if you don't return from the function.

Reply:
oh yeah. I muss oversaw it